vars/buildPlugin.groovy

```diff
@@ -55,7 +51,7 @@ def call(Map params = [:]) {
                                         '-Dmaven.test.failure.ignore',
                                 ]
                                 if (jdk.toInteger() > 7) {
-                                    /* Azure mirror only works for Java 8+ due to Letsencrypt cert */
+                                    /* Azure mirror only works for sufficiently new versions of the JDK due to Letsencrypt cert */
```

Comment:
In fact some JDK 8 builds failed, and maybe there are some private JDK 7 updates which work.